### DISCUSSION TITLE
Nesting a translation across namespaces

### DISCUSSION THREAD

**Discussion by imgalli** [i18next/i18next#1620] [Q&A]
Hi, we're currently trying to have the following translation but it doesn't seem to be working. Currently just the one language. Has anyone successfully done this?

`t('parDistance', { par: 1, number: 1 })`

namespace: app
```
{
  "parDistance": "$t(common.par) {{ par }}, {{ number }} $t(distance.yard, , {\"count\": {{ number }} } )",
}
```

namespace: units
```
{
  "distance": {
    "yard": "yd",
    "yard_plural": "yds"
  }
}
```

**Comment by jamuhl** [comment]
https://www.i18next.com/translation-function/nesting#basic -> You can reference keys from other namespaces by prepending the namespace: `"nesting1": "1 $t(common:nesting2)"`

**Reply by imgalli** [reply]
Thanks for your reply.

As part of my debug steps I moved this translation into the app namespace to see if it corrected my issue and I'm still seeing the following in the console. Note our language code is `en_GB` consistently everywhere as per Lokalise.

```
i18next.js?f6d7:22 i18next::pluralResolver: no plural rule found for: en_GB
```

**Reply by jamuhl** [reply]
languages in that form won't work

**Reply by imgalli** [reply]
Ah, is there a reason for this? I'm using https://github.com/dotcore64/i18next-fetch-backend and did the following and now it works:

```
i18n
  .use(Fetch)
  .use(initReactI18next)
  .init({
    lng: languageFromQueryParams,
    debug: ENVIRONMENT !== 'production',
    defaultNS: 'app',
    fallbackNS: ['units'],
    fallbackLng: 'en-GB',
    ...
    backend: {
      loadPath: function(lngs: string, namespaces: string) {
        return NEW_LOCALISATION_ORIGIN.replace('{{lng}}', lngs.replace('-', '_')).replace('{{ns}}', namespaces);
      },
    },
  });
```

I realise this codes brittle but gets me going in the right direction.

**Reply by jamuhl** [reply]
the reason - it's an i18next convention to use that ISO format over others...
